packages/core/src/App/Containers/Layout/header/header.jsx

```diff
@@ -3,15 +3,19 @@ import { PlatformContext } from '@deriv/shared';
 import DefaultHeader from './default-header.jsx';
 import DashboardPlatformHeader from './dashboard-platform-header.jsx';
 import DashboardHeader from './dashboard-header.jsx';
+import TradingHubHeader from './trading-hub-header.jsx';
 
 const Header = () => {
     const { is_appstore } = React.useContext(PlatformContext);
+    const { is_pre_appstore } = React.useContext(PlatformContext);
     if (is_appstore) {
         /**
          * The below line will implement when the new domain myapps.deriv.com added.
          */
         if (/myapps.deriv/.test(window.location.pathname)) return <DashboardPlatformHeader />;
         return <DashboardHeader />;
+    } else if (is_pre_appstore) {
```

Comment:
Better not to display this on logged out state and display the default header. Better check with PM or designer.

Reply:
Alright.... Will only show the trading hub header if the user is logged in and opted to use trading hub